packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -77,23 +86,44 @@ function MainLayoutHeader({
     campaignKey: ReferralCampaignKey.Search,
   });
 
+  const { currentStreak } = useReadingStreak();
+
   const renderButtons = () => {
     return (
       <>
-        <CreatePostButton
-          className={classNames(
-            'mr-0 laptop:mr-3',
-            optOutWeeklyGoal ? 'tablet:mr-0' : 'tablet:mr-3',
-          )}
-        />
+        <CreatePostButton />
+        {user && readingStreak === ReadingStreaksExperiment.V1 && (
+          <SimpleTooltip
+            interactive
+            showArrow={false}
+            visible={shouldShowStreaks}
+            container={{
+              paddingClassName: 'p-4',
+              bgClassName: 'bg-theme-bg-tertiary',
+              textClassName: 'text-theme-label-primary typo-callout',
+              className: 'border border-theme-divider-tertiary',
+            }}
+            content={<ReadingStreakPopup />}
+          >
+            <Button
```

Comment:
Should this not be part of the `ReadingStreakPopup`?

Reply:
This is the CTA. Maybe introduce a separate component `ReadingStreakButton` to contain the configurations and not make this header component messy. Wdyt?

Reply:
Hmm why are they wrapped then, feels a bit weird somehow?
Expected to have the button, and "modal/tippy" seperate from each other

Reply:
Ah, usually on modal that's the case. Though for tippy, this has been the case, to wrap the component so we can have a reference on where we are going to display the tooltip.